linera-sdk/src/test/integration/chain.rs

```diff
@@ -360,11 +361,14 @@ impl ActiveChain {
     pub async fn query<Output>(
         &self,
         application_id: ApplicationId,
-        query: impl ToBcsBytes,
+        query: impl AsRef<str>,
```

Comment:
I feel that this should be a `serde_json::Value`.

Reply:
I'm not sure if GraphQL queries are JSON though. I think they are custom strings, but we can change it later if it's not :thinking:

Reply:
They are

Reply:
Right. It's only the internal values I guess.